private-templates-service/client/src/components/Welcome/Welcome.tsx

```diff
@@ -31,9 +31,11 @@ const WelcomeContent = (props: WelcomeProps): ReactElement => {
 
   // Not authenticated, present a sign in button
   return (
-    <Button color="primary" onClick={props.authButtonMethod}>
-      Click here to sign in
-		</Button>
+    <React.Fragment>
```

Comment:
Since there is only one component, you don't need React.Fragment here.  (Fragment just allows you to return a group of components)